test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTestBase.cs

```diff
@@ -13,29 +13,29 @@ namespace Microsoft.DotNet.Tools.Builder.Tests
 {
     public class IncrementalTestBase : TestBase
     {
-        protected readonly TempDirectory _tempProjectRoot;
+        protected readonly TempDirectory TempProjectRoot;
 
-        private readonly string _testProjectsRoot;
-        protected readonly string _mainProject;
-        protected readonly string _expectedOutput;
+        protected readonly string TestProjectsRoot;
```

Comment:
Yea, I deleted it once, then thought maybe some future test class might use.
Since I doubted its worth, and you doubted its worth, it's going out :)